.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
-    "arrowParens": "avoid",
+    "arrowParens": "always",
```

Comment:
Just as a note, regarding your argument with multiple parameters: This rule affects arrow functions with single parameters only, because the parentheses are mandatory for multiple parameters.
Nevertheless Prettiers default value is "always" (didn't noticed this before) and their explanation why it's the default value seems sensible to me. So I'm totally fine with changing this rule to "always" 👍